src/server/scripts/World/go_scripts.cpp

```diff
@@ -1928,6 +1928,30 @@ class go_bells : public GameObjectScript
     }
 };
 
+/*########
+#### go_duskwither_spire_power_source
+#####*/
+
+enum DuskwitherSpirePowersource
+{
+    NPC_POWER_SOURCE_INVISIBLE_BUNNY = 17984
+};
+
+class go_duskwither_spire_power_source : public GameObjectScript
+{
+public:
+    go_duskwither_spire_power_source() : GameObjectScript("go_duskwither_spire_power_source") {}
+
+    bool OnGossipHello(Player* player, GameObject* go) override
```

Comment:
```
bool OnGossipHello(Player* /*player*/, GameObject* go) override
```

Reply:
commit out the **player** and that should fix the compile testing.

Reply:
Thanks, fixed

Reply:
You forgot to delete player so it shows as Player* /*player*/ player.
Needs to show as Player* /*player*/